### PR TITLE
Blog post: Typed Cluster API

### PR DESCRIPTION
**Pull request by patriknw**
First blog post of three in a series about the new Typed Cluster and Persistence APIs.

Full source code of examples in https://github.com/patriknw/akka-typed-blog/pull/14

### CODE REVIEW

File: blog/_posts/2017-09-28-typed-cluster.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+author: Patrik Nordwall
+series_title: Akka Typed Cluster and Persistence
+series_tag: typed
+tags: [actor,typed]
+---
+{% include JB/setup %}
+
+Akka Typed has taken a big step forward. It's now possible to use Akka Typed with Akka Cluster, Distributed Data, Cluster Sharding, Cluster Singleton, and Persistence. We have developed new typed APIs for all these features, and a new distributed registry of actor references that is the replacement of `ActorSelection` in untyped actors.
+
+These features are still using the existing, proven, implementations from the untyped modules and they require that you run with the untyped `ActorSystem` as described in [Akka Typed: Coexistence](http://akka.io/blog/2017/05/06/typed-coexistence). The alternative would have been to re-implement everything in pure Akka Typed actors, but that would have taken much longer time to complete and would become a maintenance nightmare. That said, in the long term future we might do that.
```
Comment (ktoso) [inline blog/_posts/2017-09-28-typed-cluster.md]: Maybe less dramatic wording - ` and would ... double the codebase size we'd have to maintain.`

Comment (viktorklang) [reply blog/_posts/2017-09-28-typed-cluster.md]: I'd rephrase that sentence.

" but that would have taken much longer time to complete and would become a maintenance nightmare."

" but that would have taken much longer time to complete."

Comment (patriknw) [reply blog/_posts/2017-09-28-typed-cluster.md]: ok, I can do that, but the costly part would be to maintain 2 code bases

Comment (viktorklang) [reply blog/_posts/2017-09-28-typed-cluster.md]: Yep, but the users doesn't really care about that. :)

Comment (patriknw) [reply blog/_posts/2017-09-28-typed-cluster.md]: ok :)

**Comment by patriknw** [comment]
fixed, and changed links to https
